core/embed/rust/src/ui/geometry.rs

```diff
@@ -447,7 +447,7 @@ impl Grid {
     }
 
     pub fn cells(&self, cells: GridCellSpan) -> Rect {
```

Comment:
I recall wanting to rename this to `cell_span` but apparently I didn't in my previous modifications? Please do that here.

Reply:
actually let's not do that, I will be doing a subsequent PR so i will do that there separately